src/e2e.test.ts

```diff
@@ -93,32 +94,34 @@ describe("End-to-end tests", () => {
 
   it("can differentiate between RDF and non-RDF Resources", async () => {
     const rdfResourceInfo = await fetchResourceInfoWithAcl(
-      "https://lit-e2e-test.inrupt.net/public/lit-pod-resource-info-test/litdataset.ttl"
+      `${rootContainer}lit-pod-resource-info-test/litdataset.ttl`
     );
     const nonRdfResourceInfo = await fetchResourceInfoWithAcl(
-      "https://lit-e2e-test.inrupt.net/public/lit-pod-resource-info-test/not-a-litdataset.png"
+      `${rootContainer}lit-pod-resource-info-test/not-a-litdataset.png`
     );
     expect(isRawData(rdfResourceInfo)).toBe(false);
     expect(isRawData(nonRdfResourceInfo)).toBe(true);
   });
 
   it("can create and remove empty Containers", async () => {
-    const newContainer1 = await createContainerAt(
-      "https://lit-e2e-test.inrupt.net/public/container-test/some-container/"
-    );
-    const newContainer2 = await createContainerInContainer(
-      "https://lit-e2e-test.inrupt.net/public/container-test/",
-      { slugSuggestion: "some-other-container" }
-    );
+    if (!rootContainer.includes("demo-ess")) {
+      // FIXME: An ESS bug regading PUTting containes prevents this test from passing.
+      // Once the bug is fixed, this should be cleaned up.
+      const newContainer1 = await createContainerAt(
+        `${rootContainer}container-test/some-container/`
```

Comment:
I'm assuming this is a remnant from attempting to run these against ESS anyway, but https://ldp.demo-ess.inrupt.com/105177326598249077653/test-data/container-test/ does contain `some-container`, it seems :)

Reply:
Good catch, I just removed it.